rfc/rfc023-x509credential.md

```diff
@@ -0,0 +1,453 @@
+  * `<pastype>` is the type of the holder of the certificate, always `S`.
+  * `<Abonnee-nr>` is the subscriber URA of the holder of the certificate.
+  * `<rol>` is the role of the holder of the certificate, always "0.00"
+  * `<AGB-code>` is the AGB code of the holder of the certificate.
+
+## Mapping UZI certificate to X509Credential
+
+The mapping of certificates to x509 is depending
+
+### The ROOT G3
+
+The `did:x509` specification dictates that the fingerprint of the Root CA is part of the did:x509. For mapping an UZI
+certificate to an X509Credential the ROOT CA MUST match one of the certificates in the UZI ROOT CA register hierarchy.
+For G3 this is:
+
+```asciidoc
+        ┌────────────────────────────────────┐        
+        │ Staat der Nederlanden Root CA - G3 │        
+        └────────────────┬───────────────────┘        
+                         │                            
+┌────────────────────────▼───────────────────────────┐
+│ Staat der Nederlanden Organisatie Services CA - G3 │
+└────────────────────────┬───────────────────────────┘
+                         │                            
+    ┌────────────────────▼───────────────────────┐    
+    │ UZI-register Medewerker niet op naam CA G3 │    
+    └────────────────────────────────────────────┘    
+```
+
+### Field mapping of the UZI credential
+
+The following fields are commonly used for mapping UZI certificates to X509Credentials
+* The `subject.O` the name of the holder of the certificate. Maps to `subject.O` in the X509Credential.
+* The `subject.L` The subject locality (city)
+* The `san.otherName` a string containing `<OID CA>-<versie-nr>-<UZI-nr>-<pastype>-<Abonnee-nr>-<rol>-<AGB-code>`,
+  where:
+  * `<OID CA>` is the OID of the CA that issued the certificate, `2.16.528.1.1007.99.2110` for CIBG.
+  * `<versie-nr>` is the version number of the certificate.
+  * `<UZI-nr>` is the UZI number of the holder of the certificate, same as `subject.serialNumber`.
+  * `<pastype>` is the type of the holder of the certificate, always `S`.
+  * `<Abonnee-nr>` is the subscriber URA of the holder of the certificate.
+  * `<rol>` is the role of the holder of the certificate, always "0.00"
+  * `<AGB-code>` is the AGB code of the holder of the certificate.
+
+## The use of UZI server certificate in the Nuts network or identifying organizations
+
+The focus on trust in the NUTS network for organizations lies primarily on the URA number identified as the
+`<Abonnee-nr>` on the UZI certificate. This number is used to identify the holder of the certificate within the Dutch
+healthcare ecosystem . The holder of the certificate can use the UZI certificate in combination with the private
+key to proof the ownership of the URA number. The diagram below shows how the UZI certificate can be used to transfer
+the trust from the CIBG register into the NUTS ecosystem using the `did:x509` method and the `X509Credential` Verifiable
+Credential.
+
+```asciidoc
```

Comment:
Nice diagram. Perhaps make it generic and move it to the introduction section of this spec?

Reply:
A few things which a vague for me:
- Source of Trust should be renamed to something else. I think the dutch term is "authentieke bron". I would advise to look into eidas 2 or de ARF
- Holder of Trust, I don't know what this means, is this a thing?
- Two views are mixed: the proces role (issuer, holder verifier) and compents (wallet, keys, credentials). Difficult to read.
- A dotted line looks like a flow, but why does the credential points to a VP?
- Perhaps add numbering to the steps if that is what you try to explain?